src/CodeExplorerBundle/Twig/SourceCodeExtension.php

```diff
@@ -49,7 +49,7 @@ public function getFunctions()
 
     public function showSourceCode(\Twig_Environment $twig, \Twig_Template $template)
     {
-        return $twig->render('default/_source_code.html.twig', array(
+        return $twig->render('CodeExplorerBundle::source_code.html.twig', array(
```

Comment:
My final comment before merging this change: could/should we use `@CodeExplorer/source_code.html.twig` notation instead?  Symfony still has't made an official statement, but the namespaced syntax seems to be favored for modern apps.